app-crypt/gpgme/gpgme-1.22.0.ebuild

```diff
@@ -66,7 +66,7 @@ PATCHES=(
 src_prepare() {
 	default
 
-	elibtoolize
+	eautoreconf
```

Comment:
Needs a comment for why it's autoreconf (someone might end up deleting it and replacing w/ elibtoolize or nothing if it looks like a leftover from a patch or similar).